src/components/sidebarInternals.svelte

```diff
@@ -142,49 +144,47 @@
 									>
 										{#if option.data.triplabel}
-											{#if option.data.trip_short_name}
+											{#if fixRunNumber(option.data.chateau_id, option.data.route_type, option.data.route_id, option.data.trip_short_name, option.data.vehicle_id)}
 												<span
 													style={`background-color: ${option.data.colour}; color: ${option.data.text_colour};`}
-													class="font-bold font-mono px-1 py-0.5 rounded-sm"
-													>{option.data.trip_short_name}</span
+													class="font-bold text-md px-1 py-0.5 mr-1 rounded-sm"
+													>{fixRunNumber(option.data.chateau_id, option.data.route_type, option.data.route_id, option.data.trip_short_name, option.data.vehicle_id)}</span
 												>
 											{/if}
-											{#if option.data.route_short_name}
+											{#if (option.data.route_long_name || option.data.route_short_name)}
 												<span
+													class="text-md"
 													style={`color: ${darkMode ? lightenColour(option.data.colour) : option.data.colour}`}
-													class="font-semibold"
-													>{option.data.route_short_name.replace(
-														'Counterclockwise',
-														'Anticlockwise'
-													)}</span
-												>
-											{/if}
-											{#if option.data.route_long_name}
-												<span
-													style={`color: ${darkMode ? lightenColour(option.data.colour) : option.data.colour}`}
-													>{option.data.route_long_name.replace(
-														'Counterclockwise',
-														'Anticlockwise'
-													)}</span
-												>
-											{/if}
-											{#if option.data.chateau_id == 'san-diego-mts' && option.data.route_type == 0}
-												<span class="">
-													#{option.data.vehicle_id}
+													>
+													{#if (option.data.route_long_name && option.data.route_short_name) && !(option.data.route_long_name.includes(option.data.route_short_name))}
+														<span class="font-bold">{fixRouteName(option.data.chateau_id, option.data.route_short_name, option.data.route_id)}</span>
+														<span class="font-normal ml-1">{fixRouteNameLong(option.data.chateau_id, option.data.route_long_name, option.data.route_id)}</span>
+													{:else}
+														<span class="font-semibold">{option.data.route_long_name ? fixRouteNameLong(option.data.chateau_id, option.data.route_long_name, option.data.route_id) : fixRouteName(option.data.chateau_id, option.data.route_short_name, option.data.route_id)}</span>
+													{/if}
+
 												</span>
 											{/if}
 										{:else}
 											<p>No Trip</p>
 										{/if}
 
-										{#if option.data.headsign}
-											<p>{option.data.headsign}</p>
+										<div class="my-2"></div>
+
+										{#if (option.data.headsign && option.data.headsign != option.data.route_long_name && option.data.headsign != option.data.route_short_name)}
+											<p class="text-sm">
+												Towards
```

Comment:
I'm not in favour of putting text next to the headsign like this because it is difficult to translate into all our languages we support